packages/blade/src/tokens/global/size.ts

```diff
@@ -34,6 +34,8 @@ export const size = {
   28: 28,
   /** 32 px */
   32: 32,
+  /** 33 px */
```

Comment:
We're getting _very_ specific now 😢

Reply:
Is this also on design?

Reply:
yup 😢 It needs to be exact because otherwise lines don't align